recipes/go/deactivate.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+export GOROOT="${CONDA_GOROOT_BACKUP}"
```

Comment:
Should we unset the backup variable too?

Reply:
Yes, I think we should. Will update in the next round.

Reply:
note: Done below.